RockPaperScissors/rps-challenge-apprenticeships/app.rb

```diff
@@ -0,0 +1,8 @@
+require 'sinatra/base'
+class RockPaperScissors < Sinatra::Base
+  get '/test' do
+    'test page'
+  end
+
+  run! if app_file == $0
+end
```

Comment:
Not too sure that you need this file if you've already got a controller (control.rb) file. This one looks like it doesn't really serve a purpose in the game.